virttest/utils_test/libvirt.py

```diff
@@ -824,20 +824,36 @@ def pci_label_from_address(address_dict, radix=10):
     return pci_label
 
 
-def mk_label(disk, label="msdos", session=None):
+def mk_label(disk, label="msdos", session=None, remote_vmm=None, vm_ip=None):
```

Comment:
@dzhengfy it would be great if you could move the disk related APIs to utils_disk.py.

Reply:
@balamuruhans , I will try to use the related APIs in utils_disk.py to refine this PR.

Reply:
Update